src/engine/source/kvdb/src/kvdbManager.cpp

```diff
@@ -60,6 +62,9 @@ KVDBHandle KVDBManager::loadDB(const std::string& name, bool createIfMissing)
     if (KVDB::CreationStatus::OkInitialized == result
         || KVDB::CreationStatus::OkCreated == result)
     {
+        Metrics::instance().addCounterValue("Kvdb.DatabaseCounter", 1UL);
```

Comment:
nitpick: Could you put a brief comment above the metric to know what it measures?